packages/cli/src/command-helpers/spkg.ts

```diff
@@ -0,0 +1,18 @@
+// validator that checks if spkg exists or is valid url
+
+import path from 'path';
+import { filesystem } from 'gluegun';
+
+export const isSpkgUrl = (value: string) => {
+  return value.startsWith('https://spkg.io/streamingfast');
```

Comment:
don't think we need `/streamingfast` on the subpath (for example we have semiotic on this path https://substreams.dev/semiotic-ai/uniswap-x/v0.1.0#modules)  we just need to make sure it is from `spkg.io`

we want to allow users either passing `https://spkg.io/semiotic-ai/uniswap-x-v0.1.0.spkg` or `spkg.io/semiotic-ai/uniswap-x-v0.1.0.spkg` and it should work



```suggestion
  return value.startsWith('spkg.io');
```

Reply:
Updating the check to 
```
value.startsWith(`https://spkg.io`) || value.startsWith(`spkg.io`)
```